src/Ledger/Conway/Specification/Ratify.lagda

```diff
@@ -447,27 +596,8 @@ abstract
 \end{figure*}
 
 \Cref{fig:defs:ratify-defs-i} defines the \accepted{} and \expired{}
```

Comment:
```suggestion
The \accepted{} and \expired{} functions just defined
```
(It's probably better to not refer to nearby figures in the prose anymore; these are converted by the pipeline to references to the appropriate sections, so technically they are correct, but it's weird for a sentence to refer to the section in which it lives.)